app/test/testutilsfunctions.cpp

```diff
@@ -946,3 +946,20 @@ void TestUtilsFunctions::testFormatAreaInProjectUnit()
   area2str = mUtils->formatAreaInProjectUnit( 7000, 1, project );
   QVERIFY( area2str == "1.7 ac" );
 }
+
+void TestUtilsFunctions::testOnScreenGeometryCenterToScreenCoordinates()
```

Comment:
As the method name changed in `InputUtils`
```suggestion
void TestUtilsFunctions::testRelevantGeometryCenterToScreenCoordinates()
```

Reply:
Thanks for the catch, I renamed it